engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-manage-ostree-branches.html

```diff
@@ -0,0 +1,60 @@
+<span page-title ng-model="repository">{{ 'Manage Ostree Branchesfor Repository:' | translate }} {{ repository.name }}</span>
+
+<a ui-sref="products.details.repositories.info({productId: product.id, repositoryId: repository.id})">
+  <i class="fa fa-angle-double-left"></i>
+  {{ "Back to Repository Details" | translate }}
+</a>
+
+<div data-extend-template="layouts/details-nutupane.html">
+
+  <div data-block="pane-loading"></div>
+
+  <div data-block="messages">
+    <div bst-alerts success-messages="successMessages" error-messages="errorMessages"></div>
+
+    <div bst-alert="success" ng-hide="generationTaskId === undefined">
+      <button type="button" class="close" ng-click="clearTaskId()">&times;</button>
+      <p translate>
+        Ostree Branch metadata generation has been initiated in the background.  Click
```

Comment:
general: should it be Ostree or OSTree when used in the UI?  I see a few instances of Ostree, so will not comment on each.